pilot/pkg/config/kube/crd/conversion.go

```diff
@@ -78,10 +95,14 @@ func ConvertObject(schema collection.Schema, object IstioObject, domain string)
 	if err != nil {
 		return nil, err
 	}
-	data, err := FromJSON(schema, string(js))
+	spec, err := FromJSON(schema, string(js))
 	if err != nil {
 		return nil, err
 	}
+	status, err := IstioStatusJSONFromMap(object.GetStatus())
```

Comment:
The status here is a `map`, and the Status in L120 is an interface, why do we even need to convert?